modules/react/_examples/stories/examples/MediaModal.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import {Modal} from '@workday/canvas-kit-react/modal';
+import {Box, BoxProps, Flex} from '@workday/canvas-kit-react/layout';
+import {Text} from '@workday/canvas-kit-react/text';
+import {PrimaryButton} from '@workday/canvas-kit-react/button';
+import {spaceNumbers} from '@workday/canvas-kit-react/tokens';
+
+const Image = (elemProps: React.ImgHTMLAttributes<HTMLImageElement>) => {
+  const [loaded, setLoaded] = React.useState(false);
+
+  return (
+    <Box
+      as="img"
+      display="block"
+      backgroundColor="frenchVanilla200"
+      transition="opacity ease 200ms"
+      opacity={loaded ? 1 : 0}
```

Comment:
```suggestion
      style={{opacity: loaded ? 1 : 0}}
```

Reply:
this is Nit, but there's a slight performance improvement using the style attribute!